sites/mosul/configs/openmrs/initializer_config/drugs/drugs.csv

```diff
@@ -733,7 +724,6 @@
 "bf1c407d-7463-49f0-b944-5fd00d96ef43","VACCINE POLIOMYELITIS, INACTIVATED, 0.5ml, monodose, vial","9bb33dd3-d317-4728-8ab6-c5690ef7e5fa","5ebc716e-e571-43aa-8f9b-f63a80f92cba","monodose"
 "5e44053a-a532-49fa-89ae-fca1fb2f9418","VACCINE POLIOMYELITIS, BIVALENT ORAL, 1 dose, multid. vial","37d48f81-2e9d-441c-a21e-ba8681804367","5ebc716e-e571-43aa-8f9b-f63a80f92cba","monodose, multidose"
 "9c240161-fac4-4cde-8902-fc9d62aab98f","VACCINE PNEUMOCOCCAL polysacch.. 23, monodose, 0.5ml syr.","0996f332-c0a7-43d4-8012-a06bdfaa84ca","6353ff60-1277-4e80-9f20-919e0dbfbe7e","monodose"
-"507354fe-321d-493f-888b-efee07ad9eb2","VACCINE RABIES, CCV, cell culture, monodose, vial + diluent","a8f65541-ffd4-43f7-b9ba-e967e977a78d","5ebc716e-e571-43aa-8f9b-f63a80f92cba","monodose"

```

Comment:
Are these deletions part of the screenshot shared on the ticket?

Reply:
Yes @pirupius , please have a look here: https://msf-ocg.atlassian.net/browse/LIME2-298?focusedCommentId=82585